server/core/basic_cluster.go

```diff
@@ -313,54 +401,31 @@ func (bc *BasicCluster) getWriteRate(
 
 // GetStoresLeaderWriteRate get total write rate of each store's leaders.
 func (bc *BasicCluster) GetStoresLeaderWriteRate() (storeIDs []uint64, bytesRates, keysRates []float64) {
-	return bc.getWriteRate(bc.Regions.GetStoreLeaderWriteRate)
+	bc.Regions.mu.RLock()
```

Comment:
the result of  storeLeaderWriteRate can not be accurate unless  lock or not lock.